Add condensed Promotions for V2 (#66)
elasticpath/elasticpath-dev: docs/rule-promotions/overview.md

}
``` 

## Condensed Promotion

> Condensed Promotion generally refers to the process of simplifying or aggregating multiple promotional rules into a single, more manageable rule. A condensed promotion is essentially an object that contains specific meta information about a promotion applied to the cart or order. 

Reviewer: The sentence `Condensed Promotion generally refers to the process of simplifying or aggregating multiple promotional rules into a single, more manageable rule. ` 
doesn't seem to fit here.

}
``` 

## Condensed Promotion

A condensed promotion is essentially an object that contains specific meta information about a promotion applied to the cart or order. 

> Condensed Promotion can be retrived from the following endpoints:

Reviewer 2: ```suggestion
Condensed promotions can be retrived from the following endpoints:
```

>  - [Get a Cart](/docs/carts-orders/get-a-cart#get-include-promotions): This endpoint displays cart-level condensed promotions within the included object. If `?include=items` is specified, the included object will also contain cart item-level promotions. The relationships field at the cart level will have a promotions section only when `?include=promotions` is specified.

Reviewer 2: ```suggestion
 - [Get a Cart](/docs/carts-orders/get-a-cart#get-include-promotions): This endpoint displays cart-level condensed promotions within the included object. If `?include=items` is specified, the included object also contains cart item-level promotions. The relationships field at the cart level has a promotions section only when `?include=promotions` is specified.
```
